PR_REVIEW.md

```diff
@@ -0,0 +1,15 @@
+## Summary
+We take PR review seriously. Please read https://medium.com/@mrjoelkemp/giving-better-code-reviews-16109e0fdd36#.xa8lc4i23 to understand how a PR review should be conducted. Be rational and strict in your review, make sure you understand exactly what the submitter's intent is. Anyone in the community can review a PR, but a Prebid Org member is also required. A Prebid Org member should take ownership of a PR and do the initial review.
+
+For modules and core platform updates, the initial reviewer should request an additional team member to review as a sanity check. Merge should only happen when the PR has 2 `LGTM` from the core team and a documentation PR if required.
+
+## Ticket Coordinator
```

Comment:
this still needs to be removed.

Reply:
Can you clarify the text I have to remove? Do you want us to remove (1) all the text cited here or (2) just the paragraph or (3) just Ticket Coordinator line?